monicahenderson/ninja_turtles/apps/vanishturtles/views.py

```diff
@@ -0,0 +1,20 @@
+from django.shortcuts import render, redirect, HttpResponse
+
+
+# Create your views here.
+def index(request):
+    return render(request, 'vanishturtles/index.html' )
+
+def turtle_images(request, color):
+    if color == '':
+        return render(request, 'vanishturtles/allturtles.html')
+    if color == 'blue':
+        return render(request, 'vanishturtles/leonardo.html')
+    elif color == 'orange':
+        return render(request, 'vanishturtles/michelangelo.html')
+    elif color == 'red':
+        return render(request, 'vanishturtles/raphael.html')
+    elif color == 'purple':
+        return render(request, 'vanishturtles/donatello.html')
+    else:
+        return render(request, 'vanishturtles/nohack.html')
```

Comment:
Everything looks good, but there's some repetition to think about.  Right now you have a different html file for each image even though every html file is identical.  This is a good case for just creating one single html file for the images and when you render that file, just pass the name of the image file you want to use in a context object.  For example:
if color == 'blue':
    context = {'image': 'vanishturtles/images/leonardo.jpg'}
     return render(request, vanishturtles/something.html, context)

then in the html file the image tag would look like:
     <img src="{% static image %}">